db/schema.rb

```diff
@@ -73,16 +55,6 @@
   add_index "map_user_document_user_infos", ["user_document_id"], name: "index_map_user_document_user_infos_on_user_document_id", using: :btree
   add_index "map_user_document_user_infos", ["user_information_id"], name: "index_map_user_document_user_infos_on_user_information_id", using: :btree
 
-  create_table "prototypes", force: :cascade do |t|
```

Comment:
Same as admins. I'm pretty certain this is supposed to be gone. This could be an artifact of some of the strange git commits we've seen going on.